WordPress/openverse
# Convert the report content popover to a modal

## Fixes



Related to #3582 by @obulat

## Description


This PR replaces the report content popover with a modal that matches the updated [designs in Figma](https://www.figma.com/design/chRWINXqyRzYRPZOJOJHuD/Modal-popover-for-report-and-license-definition?node-id=22-13656&m=dev).

### Previous version of the content report
<img width="300" alt="Content report popover" src="https://github.com/user-attachments/assets/90fd94a8-502f-4ba7-893a-7d4f93bf3001">

### Updated version of the content report
<img width="300" alt="Content report modal" src="https://github.com/user-attachments/assets/70b75faa-b507-47df-80b2-526d2600a193">

Note: this PR does not touch the license explanation modal because the nested modal handling requires more changes.

## Testing Instructions

1. Run the app. Go to a single result page and click on the "Report content" button. The form should open in a modal and should look/work correctly.
2. Check that the report page (e.g. http://0.0.0.0:8443/image/cfa45160-a7be-4911-9b9f-4e66678ec7f7/report) looks correctly

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
